custom/icds/location_reassignment/tasks.py

```diff
@@ -1,8 +1,31 @@
+from django.conf import settings
+from django.core.mail.message import EmailMessage
+
 from celery.task import task
 
 from custom.icds.location_reassignment.processor import Processor
 
 
 @task
-def process_location_reassignment(domain, transitions, site_codes):
-    Processor(domain, transitions, site_codes).process()
+def process_location_reassignment(domain, transitions, site_codes, user_email):
+    try:
+        Processor(domain, transitions, site_codes).process()
+    except Exception as e:
+        email = EmailMessage(
```

Comment:
@orangejenny i remember you have asked me to send emails via async task in the past but I think it was better to send this instantly, with no strong reason.